Not exactly sure how this is determined, but only public methods are
exposed over dbus.

--- a/lxsession/dbus-lxde-session.vala	2016-11-27 16:36:02.000000000 -0600
+++ b/lxsession/dbus-lxde-session.vala	2016-11-29 17:40:37.230698000 -0600
@@ -731,7 +731,7 @@ namespace Lxsession
         }
 
         /* Quit Manager */
-        private void QuitManagerLaunch()
+        public void QuitManagerLaunch()
         {
             message("Start Quit Manager");
             if (global_settings.get_item_string("Session", "quit_manager", "command") == null)
